### ISSUE TITLE
Multiple css classes on UI elements.

### ISSUE THREAD

**Issue by tristen** [mapbox/mapbox.js#271]
Marking as a todo for v1. To keep aspects of the UI dry, consistant in style, and easily extensible in theme there should be smart classes added to elements like `round` etc. I can implement this.

**Comment by jfirebaugh** [comment]
For the most part we'll be adopting the behavior and conventions of Leaflet. For controls provided by Leaflet, I don't think we want to get into overriding or adding classes outside of what Leaflet does -- though if there's something that we can make a good case for being added or changed, we'll definitely push to get it changed in Leaflet core. For our own controls, we should also stick to Leaflet's conventions, so they can be styled consistently with built-in controls.

In other words, check out the classes on stock Leaflet controls, and see if they are suitable. Do use Leaflet master, because there have been some changes since 0.5.

**Comment by tmcw** [comment]
> if there's something that we can make a good case for being added or changed, we'll definitely push to get it changed in Leaflet core

:+1: - let's never let the upstream limit what we want to build here.

**Comment by jfirebaugh** [comment]
Let's track this with #169.
